src/frontend/package.json

```diff
@@ -21,17 +21,17 @@
   },
   "dependencies": {
     "@bcgov/bc-sans": "1.0.1",
-    "@fortawesome/fontawesome-svg-core": "6.2.0",
-    "@fortawesome/free-solid-svg-icons": "6.2.0",
+    "@fortawesome/fontawesome-svg-core": "6.4.2",
+    "@fortawesome/free-solid-svg-icons": "6.4.2",
     "@fortawesome/react-fontawesome": "0.2.0",
-    "@uiw/react-md-editor": "3.19.0",
+    "@uiw/react-md-editor": "3.23.5",
     "breakpoint-sass": "3.0.0",
     "react": "18.2.0",
-    "react-datepicker": "4.8.0",
+    "react-datepicker": "4.16.0",
```

Comment:
And this. I don;t think we use this much and an html date picker should suffice for our app.